site/content/en/docs/reference/labels-and-annotations.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "Labels and Annotations"
+linkTitle: "Labels and Annotations"
+date: 2024-10-09
+---
+
+This page serves as a reference for all labels and annotations in Kueue.
+
+
+### kueue.x-k8s.io/is-group-workload
+
+Type: Annotation
+
+Example: `kueue.x-k8s.io/is-group-workload: "true"`
+
+Used on: [Workload](/docs/concepts/workload/).
+
+The label key indicates that this workload is used for a group of Pods.
+
+
+### kueue.x-k8s.io/job-completions-equal-parallelism
+
+Type: Annotation
+
+Example: `kueue.x-k8s.io/job-completions-equal-parallelism: "true"`
+
+Used on: [batch/Job](/docs/tasks/run/jobs/).
+
+The label key is used to keep `completions` and `parallelism` in sync
+
+
+### kueue.x-k8s.io/job-min-parallelism
+
+Type: Annotation
+
+Example: `kueue.x-k8s.io/job-min-parallelism: "5"`
+
+Used on: [batch/Job](/docs/tasks/run/jobs/).
+
+The label key indicates the minimum `parallelism` acceptable for the job in the case of partial admission.
+
+
+### kueue.x-k8s.io/job-uid
+
+Type: Label
+
+Example: `kueue.x-k8s.io/job-uid: "46ef6b23-a7d9-42b1-b0f8-071bbb29a94d"`
+
+Used on: [Workload](/docs/concepts/workload/).
+
+The label key in the workload resource holds the UID of the owner job.
+
+
+### kueue.x-k8s.io/managed
+
+Type: Label
+
+Example: `kueue.x-k8s.io/managed: "true"`
+
+Used on: Resources managed by Kueue.
+
+The label key that indicates an object is managed by Kueue.
+
+
+### kueue.x-k8s.io/multikueue-origin
+
+Type: Label
+
+Example: `kueue.x-k8s.io/multikueue-origin: "true"`
+
+Used on: [MultiKueue](/docs/concepts/multikueue/).
+
+The label key is used to track the creator of MultiKueue remote objects.
```

Comment:
Could you explain what is "remote objects"? I am thinking if we should say "Objects in Worker Cluster".

Reply:
Done